addons/skin.confluence/language/English/strings.po

```diff
@@ -2,6 +2,13 @@
 # Addon Name: Confluence
 # Addon id: skin.confluence
 # Addon Provider: Jezz_X, Team Kodi
+# [Consistency] Make terms, settings names, brands and other minutiae, consistent throughout file.
+# [Capitalization] Avoid capitalizing every second word. See http://grammarist.com/capitalization/
+# For example, prefer wording as "This new string" instead of "This New String".
+# [Referencing] If a suitable string already exists, reuse it, making a note of where its used!
+# [Description / location] For example, "#. Description of some setting" used on "#: xbmc/addons/guidialogaddoninfo.cpp"
+# When writing a description or setting, refer to a setting name in quotes. See existing entries for guidance.
+# For example, "Press \"OK\" for \"All seasons\"" instead of "Press OK for All seasons" after first word.
```

Comment:
This block will be overwritten by my sync utility. Please write it before the first entry in a format we have for "inter-entry" comments. Namely without space, like this. 

#[Consistency] Make terms, settings names, brands and other minutiae, consistent throughout file.

After the block, please leave one line out before the first entry (just like we do for marking empty lines.

Thanks